status-im/status-react
# [WIP] #2998: Allow both release and develop version to coexist

Unfinished attempt at #2998: "Allow both release and develop version to coexist"
Note: currently debug build fails at runtime with 
```
bot.js loading failed
console
ReferenceError: Can't find variable: require
```

### Summary:
Allow  both release and develop version to coexist on the same Android device. The debug build has package name suffix .debug, and its name is "Status Debug".

android/app/google-services.json has been moved to android/app/src/debug and .../release. The version in release has a different package_name entry.

### Steps to test:
* Build and install release build on Android.
* Do same for the debug build.
* Verify that both builds operate independently, and the debug build appears as "Status Debug" on the device.

status: wip
